packages/client-cli/test/cli-openapi-no-implementation.test.mjs

```diff
@@ -234,7 +234,7 @@ test('openapi client generation (typescript) with --types-only', async (t) => {
   }
 
   const plugin = `
-/// <reference types="./movies" />
+/// <reference types="${dir}/movies/movies" />
```

Comment:
why?

Reply:
The test was otherwise failing due to the failed compilation throwing an error now.